src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -65,12 +91,27 @@ class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
     this.setState({ showTransfer: true })
   }
 
+  onAddBundlesPress = () => {
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name